src/Microsoft.Health.Fhir.TemplateManagement/Overlay/OverlayFileSystem.cs

```diff
@@ -92,6 +98,38 @@ public void WriteBaseLayers(List<OCIArtifactLayer> layers)
             }
         }
 
+        public ManifestWrapper ReadManifest(string digest)
+        {
+            try
+            {
+                var cachePath = Environment.GetEnvironmentVariable("ORAS_CACHE");
+                var manifestContent = File.ReadAllText(Path.Combine(cachePath, "blobs", "sha256", digest));
+                return JsonConvert.DeserializeObject<ManifestWrapper>(manifestContent);
+            }
+            catch (Exception)
```

Comment:
We should catch specific errors like FileNotFoundException or JsonException here for better error handling?

Reply:
For now, we will skip using manifest if any exception occurred and using sequence number. 
Is it reasonable?